pylxd/tests/models/test_container.py

```diff
@@ -431,13 +431,15 @@ def setUp(self):
         super(TestFiles, self).setUp()
         self.container = models.Container.get(self.client, 'an-container')
 
-    def test_put(self):
-        """A file is put on the container."""
+    def test_put_delete(self):
```

Comment:
Would also need to add a test for whether the api_extension is supported and handle the negative/raise, if it's not.

Reply:
The tests are currently running against an entirely mocked LXD API as far as I can tell. I'll have to figure out how to flip the extension support.